README.md

```diff
@@ -171,3 +171,49 @@ The building process is standard. You just need to notice that the `Dockerfile`s
 DOCKER_USER=$USER ./scripts/build-docker-frontend.sh
 DOCKER_USER=$USER ./scripts/build-docker-backend.sh
 ```
+
+## Deployment
+
+This section covers the internal deployment of Substrate Telemetry to our staging and live environments.
+
+### Deployment to staging
+
+Every time new code is merged to `master`, a new version of telemetry will be automatically built and deployed to our staging environment, so there is nothing that you need to do. Roughly what will happen is:
+
+- An image tag will be generated that looks like `$CI_COMMIT_SHORT_SHA-beta`, for example `224b1fae-beta`.
+- Docker images for the frontend and backend will be pushed to the docker repo (see https://hub.docker.com/r/parity/substrate-telemetry-backend/tags?page=1&ordering=last_updated and https://hub.docker.com/r/parity/substrate-telemetry-frontend/tags?page=1&ordering=last_updated).
+- A deployment to the staging environment will be performed using these images. Go to https://gitlab.parity.io/parity/substrate-telemetry/-/pipelines to inspect the progress of such deployments.
+
+### Deployment to live
```

Comment:
I suspect it's more accurate to s/live/production/ but I could be wrong about which is the more common term.

Reply:
afaict, we used production for namings. So I think it might be good to use the same string for the documentation.

Reply:
Done!